sp/src/game/server/hl2/npc_stalker.h

```diff
@@ -15,14 +15,22 @@
 #include "entityoutput.h"
 #include "ai_behavior.h"
 #include "ai_behavior_actbusy.h"
+#ifdef EZ2
+#include "ez2/npc_husk_base.h"
+#endif
 
 class CBeam;
 class CSprite;
 class CScriptedTarget;
 
 typedef CAI_BehaviorHost<CAI_BaseNPC> CAI_BaseStalker;
 
+#ifdef EZ2
+// Stalkers use CAI_HuskSink to seamlessly integrate with husk squads
+class CNPC_Stalker : public CAI_BaseStalker, public CAI_HuskSink
```

Comment:
Are there any cases in which stalkers' existing behavior could be changed because they know inherit from the Husk Sink?

Reply:
I don't think so, at least not outside of stalkers becoming aggressive when husk squadmates are attacked.